reference/api/settings.mdx

```diff
@@ -684,13 +684,43 @@ You can use the returned `taskUid` to get more details on [the status of the tas
 
 ## Filterable attributes
 
-Attributes in the `filterableAttributes` list can be used as filters or facets.
+Attributes in the `filterableAttributes` list can be used as [filters](/learn/filtering_and_sorting/filter_search_results) or [facets](/learn/filtering_and_sorting/search_with_facet_filters).
 
 <Capsule intent="warning">
-Updating filterable attributes will re-index all documents in the index, which can take some time. We recommend updating your index settings first and then adding documents as this reduces RAM consumption.
+Updating filterable attributes will re-index all documents in the index, which can take some time. To reduce RAM consumption, first update your index settings and then add documents.
 </Capsule>
 
-[To learn more about filterable attributes, refer to our dedicated guide.](/learn/filtering_and_sorting/filter_search_results)
+### Filterable attribute object
+
+`filterableAttributes` may be either an array of strings or an array of filterable attribute objects.
+
+Filterable attribute objects must contain the following fields:
+
+| Name                    | Type             | Default value | Description                                              |
+| ----------------------- | ---------------- | ------------- | -------------------------------------------------------- |
+| **`attributePatterns`** | Array of strings | `[]`          | A list of strings indicating filterable fields                      |
+| **`features`**          | Object           | `{}`          | A list outlining filter types enabled for the specified attributes                      |
+
+#### `attributePatterns`
+
+Attribute patterns may begin or end with a * wildcard to match multiple fields: `customer_*`, `attribute*`.
+
+You may also set attributePatterns to *. In this case Meilisearch will treat all fields as filterable.
+
+#### `features`
+
+`features` allows you to decide which filter features are allowed for the specified attributes. It accepts the following fields:
+
+- `facetSearch`: Whether facet search should be enabled for the specified attributes. Boolean, defaults to `false` 
+- `filter`: A list outlining the filter types for the specified attributes. Must be an object and accepts the following fields:
+  - `equality`: Enables `=`, `!=`, `IN`, `EXISTS`, `NOT`, `AND`, and `OR`. Boolean, defaults to `true`
+  - `comparison`: Enables `>`, `>=`, `<`, `<=`, `TO`, `EXISTS`, `NOT`, `AND`, and `OR`. Boolean, defaults to `false`
+
+Calculating `comparison` filters is a resource-intensive operation. Disabling them may lead to better search and indexing performance. `equality` filters use fewer resources and have limited impact on performance.
+
+<Capsule intent="warning" title="_geo field and filter features">
+If set as a filterable attribute, `_geo` automatically activates all search features. Meilisearch will throw an error if you try to disable any `features` for a `_geo` field.
```

Comment:
````suggestion
If set as a filterable attribute, `_geo` automatically activates all geo-filtering features. The `_geo` filterable attribute doesn't interact with the `features` object and will not be matched by any `attributePatterns` like `_*`, prefer using the simple string definition for this specific case as:

```
[
  {
    "attributePatterns": [<String>, …],
    "features": { <...> }
  },
  "_geo"
]
```

This will work even in the middle of pattern definitions.
````

We don't trigger any error for `_geo`